ember.md

```diff
@@ -549,6 +549,73 @@ this.route('foo', { path: ':fooId' });
 [Example with broken
 links](https://ember-twiddle.com/0fea52795863b88214cb?numColumns=3).
 
+### Organization
+
+Routes should be grouped as follows:
+
+* Services
```

Comment:
This section should be numbered so it matches the numbering below.

Please use the following instead:

```
1. Services
1. Default route's properties
```
etc...

Reply:
Nice catch! I agree numbers are better 👍